core/amber/src/test/scala/edu/uci/ics/texera/workflow/common/storage/partition/PartitionDocumentSpec.scala

```diff
@@ -0,0 +1,3 @@
+package edu.uci.ics.texera.workflow.common.storage.partition
+
+class PartitionDocumentSpec {}
```

Comment:
Please include test cases so we know what's the expected usage.

Reply:
Done.